adcc/ReferenceState.py

```diff
@@ -155,7 +155,9 @@ def __init__(self, hfdata, core_orbitals=None, frozen_core=None,
             self.orbital_coefficients, self.conv_tol
         )
 
-        self.addons = hfdata.addon_functions
+        if hasattr(hfdata, "excitation_energy_corrections"):
+            self.excitation_energy_corrections =\
+                hfdata.excitation_energy_corrections
```

Comment:
```python
for name in ["excitation_energy_corrections"]:
    if hasattr(hfdata, name):
         setattr(self, name, getattr(hfdata, name))
```

Reply:
Think towards the future ;)

Reply:
Sure, good idea. I was just trying to fix the existing tests...